core/client-internal/src/main/java/tachyon/client/ClientContext.java

```diff
@@ -42,85 +38,61 @@
   private static TachyonConf sTachyonConf;
   private static InetSocketAddress sMasterAddress;
   private static ClientMetrics sClientMetrics;
-  private static boolean sInitialized;
 
   static {
-    sInitialized = false;
     reset();
   }
 
   /**
-   * Initializes the client context singleton.
+   * Resets to the default Tachyon configuration and initializes the client context singleton.
    */
-  public static synchronized void reset() {
-    reset(new TachyonConf());
+  private static void reset() {
```

Comment:
is it necessary to still keep a private `reset()`? I feel `reset()`  should probably be replaced (or merged with) by `init()`, can we?